apps/rush-lib/src/cli/actions/ChangeAction.ts

```diff
@@ -64,6 +64,11 @@ export default class ChangeAction extends BaseRushAction {
         'PATCH - these are changes that are backwards and forwards compatible. ' +
         'Examples are: Modifying a private API or fixing a bug in the logic ' +
         'of how an existing API works.',
+        '',
+        'HOTFIX (EXPERIMENTAL) - these are changes that are hotfixes targeting a ' +
+        'specific older version of the package. When a hotfix change is added, ' +
+        'other changes will not be able to increment the version number.', +
+        'Enabled this feature by setting \'hotfixChangeEnabled\' in your rush.json.', +
```

Comment:
How do you prevent other changes from happening to the package?

Can you throw errors if it happens?

Reply:
I think it should throw errors if you try and publish a hotfix change when there are also patch/minor/major changes. And you should not try to make patch/minor/major changes on top of a prerelease branch.

Reply:
Tests were added for applying a patch and hotfix on the same package. (should error)